GoogleUtilities/AppDelegateSwizzler/GULAppDelegateSwizzler.m

```diff
@@ -882,17 +883,30 @@ - (void)application:(GULApplication *)application
       didReceiveRemoteNotificationWithCompletionIMP =
           [didReceiveRemoteNotificationWithCompletionIMPPointer pointerValue];
 
+  dispatch_group_t __block callbackGroup = dispatch_group_create();
+  UIBackgroundFetchResult __block latestFetchResult = UIBackgroundFetchResultNoData;
+  dispatch_group_notify(callbackGroup, dispatch_get_main_queue(), ^() {
```

Comment:
It looks like calling `dispatch_group_notify` on an empty group will lead to scheduling the block straight away. I think it should be moved to after line 916.